testing_utils_test.go

```diff
@@ -54,6 +59,20 @@ type restMethod struct {
 	Version string `yaml:"version,omitempty"`
 }
 
+// serverConfig holds the Marathon server configuration
+type serverConfig struct {
+	// The version of Marathon
+	version string
+}
+
+// configContainer holds both server and client Marathon configuration
+type configContainer struct {
+	client   *Config
+	server   *serverConfig
+	username string
+	password string
```

Comment:
Unless I'm mistaken, username and password are only used on the (fake) server side. They should thus live in `config.ServerConfig`.